operations/iuf/workflows/management_rollout.md

```diff
@@ -496,7 +497,19 @@ Once this step has completed:
 Return to the procedure that was being followed for `management-nodes-rollout` to complete the next step, either [Management-nodes-rollout with CSM upgrade](#31-management-nodes-rollout-with-csm-upgrade) or
 [Management-nodes-rollout without CSM upgrade](#32-management-nodes-rollout-without-csm-upgrade).
 
-## 4. Update management host Slingshot NIC firmware
+## 4. Restart `goss-servers` on all NCNs
+
+The `goss-servers` service needs to be restarted on all NCNs. This ensures the correct tests are run on each NCN. This is necessary due to a timing issue in the NCN's cloud-init which has been fixed in CSM 1.6.1.
```

Comment:
Shouldn't it be "which is going to be fixed in CSM 1.6.1" ?

Reply:
We aren't going to want to make another PR to change the verb tense after 1.6.1 is released. But we do probably want to start these paragraphs with something that indicates it only applies if they are on a version lower than 1.6.1.

```suggestion
## 4. Restart `goss-servers` on all NCNs

> Skip this step if the CSM version is 1.6.1 or above.

If the CSM version is lower than 1.6.1, then the `goss-servers` service needs to be restarted on all NCNs. This ensures the correct tests are run on each NCN. This is necessary due to a timing issue in the NCN's cloud-init which is fixed in CSM 1.6.1.
```

Reply:
I am going to leave this wording "which has been fixed in CSM 1.6.1" so that we do not need to change the verb tense with another PR later.